1.4/Patches/MSSMeme_SpeakUp_Romance_Patch.xml

```diff
@@ -0,0 +1,289 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+    <Operation Class="PatchOperationFindMod">
+        <mods>
+            <li>SpeakUp</li>
+        </mods>
+        <match Class="PatchOperationSequence">
+            <operations>
+                <li Class="PatchOperationReplace">
+		<xpath>/Defs/InteractionDef[defName = "RomanceAttempt"]/logRulesInitiator/rulesStrings</xpath>
+		<value>
+		  <rulesStrings>
+			<li>r_logentry->[attempt]</li>
+				<li>attempt(INITIATOR_relationship==Lover,tag=RomanceSuccess)->[try_romance]</li>
+				<li>attempt(priority=-1,tag=RomanceFail)->[try_romance]</li>
+
+				<li>try_romance->I drew you as a waifu.</li>
+				<li>try_romance->Your [romantic_bodypart] is gorgeous, would you like to be my reference for this drawing?</li>
+				<li>try_romance->Your [romantic_bodypart] makes me forget about my OC's dead parents.</li>
+				<li>try_romance->Could you pose for my artwork .... in private?</li>
+				<li>try_romance->Can we swap [romantic_artrelation] collections?</li>
+				<li>try_romance->I think our original characters would look really good together. But perhaps... as more than friends.</li>
+				<li>try_romance->I did a few of doodles of you in my sketchbook, would you like to see all 10 pages!?</li>
+				<li>try_romance->I'd love it if you'd get naked and lie down like a pancake on that canvas so I can trace out your curves.</li>
+				<li>try_romance->You look just like my mentally deranged OC. Want to get freaky?</li>
+				<li>try_romance(INITIATOR_trait==bloodlust)->*Stabs pencil through hand to get attention* UwU.</li>
+				<li>try_romance(INITIATOR_trait==bloodlust,RECIPIENT_artistic_passion!=None)->The pattern of blood and viscera that stains the ground after I fight in a battle is like a fine tapestry to me. If you and I were side by side, we could create wondrous masterpieces that would put even the finest artists to shame.</li>
+
+
+			<li>romantic_bodypart->hand</li>
+			<li>romantic_bodypart->leg</li>
+			<li>romantic_bodypart->face</li>
+			<li>romantic_bodypart->eye</li>
+
+			<li>romantic_artrelation->yaoi</li>
+			<li>romantic_artrelation->yuri</li>
+			<li>romantic_artrelation->fursona</li>
+
+
+				<!-- ===passion-based art seduction=== -->
+
+
+				<li>try_romance(RECIPIENT_shooting_passion!=None)->I know you like to go hunting, so I drew you slaying a [animal_grand] with a well placed shot [vital_bodypart]! Do... do you like it?</li>
```

Comment:
Maybe vary the do... do you like it a little?

`What do you think...`
`Did... did I do well`
`Does it... spark joy`